### PR TITLE
Launch Dancing Snacks Feature

### PR DESCRIPTION
**Pull request by slcrider**
The `Dancing Snacks` feature is an interactive Processing sketch that utilizes a snacks sprite sheet to fill your screen with cute, yummy snacks. To interact, the user has the following commands:
- Pressing **spacebar** results in the addition of a snack
- Pressing **d** deletes the first snack in the snacks list
- Pressing **esc** ends the program

For additional flair, a splash screen was created for display on program load until the user adds a snack. Once a snack is added, the user will be in the core snacks screen, featuring a Miami Vice-esqe color gradient and cheesy 80's font "Snacks!" test.

Resources for this project:
- Splash Screen: Jake Jones
- Sprite Sheet of Snacks: neoriceisgood on deviantart, [available here](https://orig00.deviantart.net/4b9f/f/2013/015/d/3/100_food_sprites_by_neoriceisgood-d5rl3l4.png).

### CODE REVIEW

**Comment by slcrider** [comment]
There are a lot of commits here. The squash-and-merge option might not be a bad one.

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,6 @@
+<<<<<<< HEAD
```
Comment (colvinwellborn) [inline .gitignore]: remnants of a conflict, should be removed

File: .gitignore [hunk 2]
```diff
@@ -58,3 +58,4 @@ Session.vim
 tags
 
 # End of https://www.gitignore.io/api/linux,vim,macos
+>>>>>>> d1dacc915ff556483f04d5ae66d980762071a0f1
```
Comment (colvinwellborn) [inline .gitignore]: and this one

**Review by colvinwellborn — APPROVED** [review]
SNAXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXZ

**Pull request merged**